### PR TITLE
Allow us to link directly to speakers' dialogs

### PR DESCRIPTION
**Pull request by arsduo**
We want to be able to post tweets with links directly to the info about a speaker's talk -- this PR does that!

It'd be nice to have a URL like `/speaker/NAME`, but that doesn't work (at least in local dev) because of how we're serving the app. Instead, we use a query parameter.

**Test steps**

Try it out!

http://localhost:5000/?speaker=BrianHicks
http://localhost:5000/?speaker=KatieHughes
http://localhost:5000/?speaker=RichardFeldman

Note that the query parameter remains even if you close the dialog; IMO that's okay, but we can fix that if desired.

### CODE REVIEW

**Comment by arsduo** [comment]
speaker names

File: package-lock.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
       "integrity": "sha1-63d99gEXI6OxTopywIBcjoZ0a9I=",
```
Comment (carolineartz) [inline package-lock.json]: why all the updates here do youo know?

Comment (arsduo) [reply package-lock.json]: No idea but I'll revert them before merging

Comment (arsduo) [reply package-lock.json]: Reverted

**Comment by carolineartz** [comment]
@arsduo i think we should just do the modal speaker population through routing. That takes care of the modal opening / closing associating with the presence of the query params -- so when you open the modal from the site it will have the params you could grab that direct link. Closing the modal would automatically remove the query params, so that would be out of the way too... do you want me to push something to another branch os you can see what i mean?

**Comment by arsduo** [comment]
I see what you mean — I can hack on that a bit tonight most likely, unless
you want to do it.

The main problem is that `npm serve` doesn't seem to support multiple pages
(at least by default), so we'll need to solve that.

**Comment by carolineartz** [comment]
im not sure what you meana about the multple pages thing? either way i didn't mean making it a separate page (cuz for SEO reasons and how netlify serves we have separate html files for each paage)... i just meant using routing to handle in-app linking to query params. so basically just like a separate route for that view. Ultimately we should add fragments that link to the speaker sections too but for another day. 

We'll get rid of the `ClickOpen...`... `ClickClose...` messages and justs hook into the elm navigation 
https://github.com/ryannhg/elm-in-the-spring/compare/speaker-link-routes?expand=1

**Comment by arsduo** [comment]
Sorry, multiple routes. Anything but / isn't served by our current
configuration.

**Comment by carolineartz** [comment]
handled in #8

**Pull request closed without merging**